services/app-api/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "extends": "../../tsconfig.json",
     "compilerOptions": {
         "module": "commonjs",
-        "lib": ["es2021"],
+        "lib": ["es2021", "DOM"],
```

Comment:
this feels... unexpected? What this still needed or is it from previous debugging

Reply:
The s3 APIs return a ReadableStream which is a browser type. Without this the types from the AWS api were coming back without the right information. In the end we’re not actually using that type, but I think we should keep this so the types stay valid